src/community/who-uses-dart.md

```diff
@@ -4,23 +4,88 @@ description: Mobile apps and Google web apps are two of the most common uses for
+    and still loaded fast, they turned to Flutter."
+* [PUBG][]: 
+    > "When the PUBG MOBILE team wanted to add a community module to allow players
+    from all over the world to share gameplay clips and more, they turned to Flutter."
+
+[Flutter toolkit]: {{site.flutter}}
+[Flutter showcase]: {{site.flutter}}/showcase
+[BMW]: https://www.press.bmwgroup.com/global/article/detail/T0328610EN/the-my-bmw-app:-new-features-and-tech-insights-for-march-2021?language=en
+[Sonos]: https://tech-blog.sonos.com/posts/renovating-setup-with-flutter/
+[Alibaba]: https://flutter.dev/showcase/alibaba-group
+[PUBG]: https://flutter.dev/showcase/pubg-mobile
+
+## Google 
+
+Dart is used in almost every product group at Google:
+
+* The [Google Play Console][] lets app publishers market Android apps, and
+is written completely in Dart.
+* The [Google Ads][] team continues to grow, with hundreds of SWEs primarily
+writing Dart code.
+* The [Google Assistant][] team uses Flutter and Dart to drive the overall system
+UI of their Smart Displays.
+* Both the UI and much of the back-end code for [Google Nest Hub][] and Hub Max
+devices are written in Dart.
+* The list goes on, with many more Google applications made using Dart and Flutter:
+  * [Google Pay][]
+  * [Google One][]
+  * [Google Fibre][]
+  * [Google Elections][]
+  * [Google Shopping][]
+  * [Cloud Search][]
+  * [Family Link][]
+  * [Google One][]
+  * [Cloud Search][] 
+
+Several other Google teams are currently rewritting their projects in Flutter.
+In all, Google has thousands of developers writing millions of lines of code in Dart.
+
+[Google Play Console]: https://android-developers.googleblog.com/2020/06/introducing-new-google-play-console-beta.html
 [Google Ads]: https://ads.google.com/getstarted
-[flutter-io19]: https://developers.googleblog.com/2019/05/Flutter-io19.html
+[Google Assistant]: https://developers.googleblog.com/2019/05/Flutter-io19.html
+[Google Nest Hub]: https://store.google.com/us/product/nest_hub_2nd_gen?hl=en-US
+[Google Pay]: https://pay.google.com/intl/en_in/about
+[Google Fibre]: https://fiber.google.com
+[Google Elections]: https://elections.google
+[Google Shopping]: https://shopping.google.com
+[Family Link]: https://families.google/familylink
+[Google One]: https://one.google.com/about
+[Cloud Search]: https://workspace.google.com/intl/en_in/products/cloud-search
+
+## External
+
+Beyond Google and Flutter, many companies use Dart for building their applications:
+
+* [Workiva's][] next-generation client applications for their productivity suite
+are built with Dart.
+* [Wrike][] uses the [AngularDart][] framework to develop project management tools.
+* [Blossom][] is an agile project management tool built with Dart.
+* And many more!
+
+If you build production software with Dart and want to be included in this list,
+please visit [our site's GitHub] to open an issue or pull request, thanks!
```

Comment:
**issue:** Strike `please` per the [GSG](https://developers.google.com/style/tone#politeness-and-use-of-please). Also, use `GitHub` as an adjective describing the repo, per the [GSG](https://developers.google.com/style/possessives#product,-feature,-and-company-names).
```suggestion
visit the [Dart.dev GitHub repo] to open an issue or pull request.
```